networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,73 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-operator-1.4.0-bug-fixes"]
+=== Bug fixes
+* Previously, the Prometheus metrics exported by Network Observability were computed out of potentially duplicated network flows. In the related dashboards, from *Observe* -> *Dashboards*, this could result in potentially doubled rates. Note that dashboards from the *Network Traffic* view were not affected. Now, network flows are filtered to eliminate duplicates prior to metrics calculation, which results in correct traffic rates displayed in the dashboards. (link:https://issues.redhat.com/browse/NETOBSERV-1131[*NETOBSERV-1131*])
+
+* Previously, the Network Observability Operator agents were not able to capture traffic on network interfaces when configured with Multus or SR-IOV, non-default network namespaces. Now, all available network namespaces are recognized and used for capturing flows, allowing capturing traffic for SR-IOV. There are xref:../../networking/network_observability/configuring-operator.adoc#network-observability-SR-IOV-config_network_observability[configurations needed] for the `FlowCollector` and `SRIOVnetwork` custom resource to collect traffic. 
+(link:https://issues.redhat.com/browse/NETOBSERV-1283[*NETOBSERV-1283*])
+
+* Previously, in the Network Observability Operator details from *Operators* -> *Installed Operators*, the `FlowCollector` *Status* field might have reported incorrect information about the state of the deployment. The status field now shows the proper conditions with improved messages. The history of events is kept, ordered by event date. (link:https://issues.redhat.com/browse/NETOBSERV-1224[*NETOBSERV-1224*])
+
+* Previously, during spikes of network traffic load, certain eBPF pods were OOM-killed and go into `CrashLoopBackOff` state with the 429 error: 'Per stream rate limit exceeded (limit:xMB/sec) while attempting to ingest for stream'. Now, you can xref:../../networking/network_observability/troubleshooting-network-observability.adoc#network-observability-troubleshooting-loki-tenant-rate-limit_network-observability-troubleshooting[configure the `LokiStack` custom resource `spec.limits.global.ingestion`] with the `perStreamRateLimit` and `perStreamRateLimitBurst` specifications restart the pods in your cluster and stop the 429 rate-limit error. (link:https://issues.redhat.com/browse/NETOBSERV-975[*NETOBSERV-975*]) 
+
+[id="network-observability-operator-1.4.0-known-issues"]
+=== Known issue
+* Since the 1.2.0 release of the Network Observability Operator, using Loki Operator 5.6, a Loki certificate change periodically affects the `flowlogs-pipeline` pods and results in dropped flows rather than flows written to Loki. The problem self-corrects after some time, but it still causes temporary flow data loss during the Loki certificate change. This issue has only been observed in large-scale environments of 120 nodes or greater. (link:https://issues.redhat.com/browse/NETOBSERV-980[*NETOBSERV-980*])
```

Comment:
should we list [DNS TCP issue](https://issues.redhat.com/browse/NETOBSERV-1245) under known issue? it was reported as Bug but later converted as story given the scope. I feel it'd be useful for customer to learn about this limitation with DNSTracking feature cc @msherif1234

Reply:
also, how about listing mDNS also something not supported yet?

Reply:
@memodi I don't see mDNS listed in the API as "unsupported"...I guess usually we reserve those unsupported mentions for "unsupported but still works at your own risk" type things. For example, IPFIX was listed as "unsupported" for a while. My feeling is we don't usually mention unsupported things in the release notes, unless its a deprecation notice.

Reply:
yeah we need to say no mDNS, no DNS over TCP and no DNS-SD "service discovery"

Reply:
@skrthomas - this isn't about something that already existed, but more about other DNS features that may imply to be working/supported while we know with 1.4 they do not work (existing reported issues), so would be better to list them.

Reply:
Ok i added two known issues for https://issues.redhat.com/browse/NETOBSERV-1304 and https://issues.redhat.com/browse/NETOBSERV-1245. Thanks for adding the RN Text n the jiras @msherif1234